app/controllers/application_controller.rb

```diff
@@ -8,6 +8,8 @@ class ApplicationController < ActionController::Base
 protected
 
   def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: %i[given_name family_name roles organisation])
```

Comment:
can we drop the organisation attribute (across the codebase)? the recently provisioned cognito doesn't have that and currently we have no use for it...